book/scheduling.md

```diff
@@ -689,15 +686,15 @@ class JSContext:
         self.tab.set_needs_render()
 ```
 
-There are more calls to `render`; you should find and fix all of them...
-Except, let's take a closer look at `click`.
+There are more calls to `render`; you should find and fix all of
+them...except, let's take a closer look at `click`.
```

Comment:
Haha, I think I just changed this the other way. That's fine, let's do it your way and see if the copyeditor complains.